packages/front-end/pages/oauth/lookup.tsx

```diff
@@ -42,33 +41,63 @@ export default function OAuthLookup() {
 
   const [open, setOpen] = useState(true);
 
-  if (!open) {
-    return <LoadingOverlay />;
-  }
-
+  const leftside = (
+    <>
+      <h1 className="title h1">GrowthBook Enterprise</h1>
+      <p></p>
+    </>
+  );
   return (
-    <Modal
-      open={true}
-      autoCloseOnSubmit={false}
-      submit={form.handleSubmit(async ({ email }) => {
-        // This will lookup the SSO config id and store it in a cookie
-        await lookupByEmail(email);
-        await redirectWithTimeout(window.location.origin);
-      })}
-      close={async () => {
-        setOpen(false);
-        await safeLogout();
-      }}
-      closeCta="Cancel"
-      cta="Continue"
-    >
-      <h3>Enterprise SSO Login</h3>
-      <p>
-        Enter your email address and you will be redirected to your
-        company&apos;s SSO portal, if configured.
-      </p>
-      <Field label="Email Address" {...form.register("email")} type="email" />
-    </Modal>
+    <WelcomeFrame leftside={leftside} loading={!open}>
+      <form
+        onSubmit={form.handleSubmit(async ({ email }) => {
+          // This will lookup the SSO config id and store it in a cookie
+          await lookupByEmail(email);
+          await redirectWithTimeout(window.location.origin);
```

Comment:
This needs a loading state and error handling.  The `<Modal>` component was providing that before, but we need to add it manually here if we're not using that.

Reply:
Done.